resources/views/UserDashboard.blade.php

```diff
@@ -1,20 +1,195 @@
+<?php 
+	$threadTrends = [
+	 	 	[1,1,[1,1,1]],//sub array is for exercises in the week 1=>completed, 0=> not completed
+	 	 	[2,1,[1,0]],
+	 	 	[3,0,[0]],
+	 	 	[4,0,[0]]
+	 ];
+	 $threadTrends = (array) $threadTrends;
+
+	 $weeksExp = [
+ 		'week 1'=>[
+	 			['title'=>'Phy 107', 'experimentType'=>'Experiment I', 'start'=>'11-12-2020','submit'=>'11-12-2020','page'=>'simple-pendulum','status'=>1],
+	 			['title'=>'Phy 107', 'experimentType'=>'Experiment II', 'start'=>'11-12-2020','submit'=>'11-12-2020','page'=>'vernier-caliper','status'=>1],
+	 			['title'=>'Phy 107', 'experimentType'=>'Experiment III', 'start'=>'11-12-2020','submit'=>'11-12-2020','page'=>'micrometer','status'=>1],
+ 		],
+ 		'week 2'=>[
+	 			['title'=>'Phy 107', 'experimentType'=>'Experiment I', 'start'=>'11-12-2020','submit'=>'11-12-2020','page'=>'micrometer','status'=>1],
+	 			['title'=>'Phy 107', 'experimentType'=>'Experiment II', 'start'=>'11-12-2020','submit'=>'11-12-2020','page'=>'vernier-caliper','status'=>0]
+ 		],
+ 		'week 3'=>[
+	 			['title'=>'Phy 107', 'experimentType'=>'Experiment I', 'start'=>'11-12-2020','submit'=>'11-12-2020','page'=>'simple-pendulum','status'=>0]
+ 		],
+ 		'week 4'=>[
+	 			['title'=>'Phy 107', 'experimentType'=>'Experiment I', 'start'=>'11-12-2020','submit'=>'11-12-2020','page'=>'micrometer','status'=>0]
+ 		]
+	 ];
+	 $weeksExp = json_decode(json_encode($weeksExp));
+ ?>
 @extends('layouts/main')
+@section('head')
+<script src="{{ asset('js/jquery-1.11.3.min.js')}}"></script>
+
+<script>	
+	 setInterval(function() {
+	 	if ($(window).width() <1000) {
+	 		$('.toberemoved').hide();
+	 	}else{
+	 		$('.toberemoved').show();	 		
+	 	}
+	 }, 1000);
+		//console.log(<?php// echo json_encode($weeksExp);?>);
+</script>
+
+@endsection
 @section('content-body')
-    <div class="dashboad-content" style="display: flex;">
-     
-
-   <v-msidebar home="{{route('home')}}" explore="{{route('explore')}}" settings="{{route('home')}}" mycourse="{{route('my-course-review')}}" courses="{{route('courses')}}" incourse="1" active="mycourse"  ></v-msidebar>   
-        <div class="dashboard-content" style="width: 100%;">
-         <!-- top nav bar -->
-            <v-topnav></v-topnav>
-         <!-- end to nav bar -->
-
-       
-            <v-lactivity></v-lactivity>
-       
-            <v-practicaltimeline></v-practicaltimeline>
-        
-
-        </div>
-    </div>
-@endsection
\ No newline at end of file
+<div style="display: flex;">   
```

Comment:
@softdiddy Wondering why this changes? Isn't that part of https://github.com/cs-dept-ibbul/vlab/pull/27?

Reply:
No changes, may be its just space or something